src/elements/table/table-wrapper/table-wrapper.scss

```diff
@@ -4,26 +4,18 @@
 @include sbb.box-sizing;
 
 :host {
-  --sbb-table-wrapper-border-radius: var(--sbb-border-radius-4x);
-
   display: block;
+  overflow: auto;
 }
 
-::slotted(.sbb-table) {
-  width: 100%;
+:host(:not(.sbb-table-wrapper-offset-none)) {
+  padding-bottom: var(--sbb-spacing-fixed-1x);
 }
 
-.sbb-table-wrapper {
-  height: 100%;
-  overflow: auto;
-  @include sbb.scrollbar($size: thick, $track-visible: true);
-
-  :host([negative]) & {
-    @include sbb.scrollbar-variables--color-negative;
-    @include sbb.scrollbar-variables--color-negative-track-visible;
-  }
+:host(:focus-visible) {
+  @include sbb.focus-outline;
+}
```

Comment:
In negative mode focus outline is still black, making it not visible

Reply:
That is on purpose, as the outline is around the negative table. the negative table can be used inside a positive context. the outline is depending on the background of the surrounding area and therefore is responsibility of consumer or outer component.